dms_storage/models/dms_storage.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 CreuBlanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class DmsStorage(models.Model):
+    _inherit = "dms.storage"
+
+    save_type = fields.Selection(
+        selection_add=[("storage", "Storage")], ondelete={"storage": "cascade"}
```

Comment:
This is really necessary? With this approach, it could not be used as an attachment type storage actually.

Reply:
You can setup the attachments to be stored on a storage backend, so it doesn't seem to restrict any functionality.